src/utils/getFilenameFromUrl.js

```diff
@@ -95,8 +95,15 @@ function getFilenameFromUrl(context, url) {
         filename = path.join(outputPath, querystring.unescape(pathname));
       }
 
-      let fsStats;
+      if (
+        context.outputFileSystem.existsSync &&
+        !context.outputFileSystem.existsSync(filename) &&
+        options.historyApiFallback
+      ) {
+        filename = path.join(outputPath);
```

Comment:
Why do we need `path.join` here, it doesn't have sense

Reply:
I will check. Look like mistake. May be we need change it to path.resolve

Reply:
I think it is required.
<img width="425" alt="image" src="https://user-images.githubusercontent.com/1561460/224569989-17ac8b8d-1c65-4a28-a74c-3dbc3c4a0a2b.png">

May be i can remove condition?

Reply:
Now this code looks like this option is not necessarily at all.

Reply:
Everything should work without this option.
I am confused. 
I feel that i need cancel this PR.